kolibri/auth/backends.py

```diff
@@ -21,7 +21,7 @@ def authenticate(self, username=None, password=None, facility=None):
         :param facility: a Facility
         :return: A FacilityUser instance if successful, or None if authentication failed.
         """
-        users = FacilityUser.objects.filter(username=username)
+        users = FacilityUser.objects.filter(username__iexact=username)
```

Comment:
Makes sense since we don't allow users to create the same case insensitive username (When we serialize/create we validate the case insensitive username). Should we just be storing all usernames as case insensitive in the backend at this point? or is there a strong reason to have the original usernames as they were inputted?